### DISCUSSION TITLE
Protein quant Vs peptide quant

### DISCUSSION THREAD

**Discussion by witszymanski** [vdemichev/DiaNN#213] [General]
Hi all!
I came across an issue today, and I am not sure how to bite it. When I look at the **pg_matrix** I can see that my protein is almost in every sample, also samples that have this protein knocked out (marked in red):
![image](https://user-images.githubusercontent.com/15866068/137300295-3b309da2-c15d-4e84-bb4f-20350e5fb097.png)

But when I look at the **pr_matrix** then it is quite clear, that this protein is rather missing. I have almost no peptides, in comparison to other samples. Still i can see, that DIA-NN takes an average of peptide signal, and not the sum, so it doesn't matter if there is only 1 or 10 peptides found. 
![image](https://user-images.githubusercontent.com/15866068/137300689-1460f544-706b-47be-95c3-c77f0f95d449.png)

You see here in this comparison, that protein intensity pattern from **pg_matrix** is similar to the average of peptides from pr_matrix
![image](https://user-images.githubusercontent.com/15866068/137301073-800ff917-b83b-4644-bc38-00d144da6336.png)

Anyone can comment on this? Is there a setting that I missed which would fix that? 

![image](https://user-images.githubusercontent.com/15866068/137301266-baf5c85a-ccad-4754-b7b3-e197266afc99.png)

Greets!
Witek

**Comment by vdemichev** [comment]
Hi Witek,

Please see about false discovery rates & protein q-values here https://github.com/vdemichev/DiaNN#frequently-asked-questions. By default the pg_matrix file is filtered using global protein q-value. You probably want run-specific filtering. For this, use --matrix-spec-q, to apply run-specific protein q-value to matrices, or, alternatively, just filter the way you like using the main report produced by DIA-NN.

Best,
Vadim
